mmedit/models/base_archs/__init__.py

```diff
@@ -19,6 +23,35 @@
 from .upsample import PixelShufflePack
 from .vgg import VGG16
 
+
+def register_diffusers_models() -> List[str]:
+    """Register models in ``diffusers.models`` to the ``MODELS`` registry.
```

Comment:
If we are referring to the `diffusers` library of huggingface, the docstring should be clearer.

Reply:
```suggestion
"""  Register models in ``diffusers.models`` to the ``MODELS`` registry.
    Specifically, the registered models from diffusers only defines the 
    network forward without training. 
    See more details about diffusers in: https://github.com/huggingface/diffusers
```